src/BizHawk.Client.EmuHawk/BizHawk.Client.EmuHawk.csproj

```diff
@@ -85,6 +85,7 @@
 	</ItemGroup>
 -->
 	<ItemGroup>
+		<PackageReference Include="DiscordRichPresence" Version="1.0.150" />

```

Comment:
This is a third-party wrapper over the first-party C/C++ lib. Not necessarily a bad thing, I'm just personally wary of adding more technical debt. Also, it targets .NET Standard 2.0.

---

Not sure where to leave this comment, but it'd be nice if this feature was shared between front-ends. Sadly we don't have a good abstraction for that yet, so if this was all moved to `Client.Common`, there wouldn't be an obvious place for it.